packages/icon/package.json

```diff
@@ -25,7 +25,9 @@
     "@leafygreen-ui/palette": "^4.0.0"
   },
   "dependencies": {
-    "@leafygreen-ui/emotion": "^4.0.3"
+    "@leafygreen-ui/emotion": "^4.0.3",
+    "@leafygreen-ui/button": "^20.0.2",
```

Comment:
icon probably should not rely on button and typography? i think this is introducing a circular dep bc button has a devdep on icon?